public/editor/scripts/main.js

```diff
@@ -53,7 +45,7 @@ function init(BrambleEditor, Project, SSOOverride, ProjectRenameUtility) {
   });
 }
 
-require(["jquery", "bowser"], function($, bowser) {
+require(["jquery", "bowser", "bramble-editor", "project", "sso-override", "fc/project-rename"], function($, bowser, BrambleEditor, Project, SSOOverride, ProjectRenameUtility) {
```

Comment:
This might be OK now, but before we had cases where IE was refusing to load `bramble.js`, which mean that `Bramble` wasn't defined globally, so it would fail and not run our checks on startup.  If you do this, you need to change how we access `Bramble`, probably doing a check on `window.Bramble` and then if it's missing, we need to call the generic error provider here.

What about the idea from https://github.com/mozilla/thimble.webmaker.org/issues/1035#issuecomment-137204974 of putting these modules in the `include` of the build?